library/std/src/io/impls.rs

```diff
@@ -306,6 +306,10 @@ impl BufRead for &[u8] {
 ///
 /// Note that writing updates the slice to point to the yet unwritten part.
 /// The slice will be empty when it has been completely overwritten.
+///
+/// If the amount to be written exceeds the size of the slice, write operations will return short
```

Comment:
```suggestion
/// If the amount of bytes to be written exceeds the size of the slice, write operations will return short
```

Reply:
I will write "number of bytes" since I think "amount of bytes" isn't proper grammar :-).